dojo/templates/dojo/dismiss_announcement.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
```

Comment:
```suggestion
{% extends "base.html" %}
{% load i18n %}
```

Dismissing an announcement currently throws a 500 for me. Need to load internationalization from Django to use translation tag.